ladybug/debugger/src/main/java/org/frankframework/ladybug/PipeLineSessionDebugger.java

```diff
@@ -32,14 +32,14 @@
 public class PipeLineSessionDebugger implements MethodHandler {
 
 	private final PipeLineSession pipeLineSession;
-	private final IbisDebugger ibisDebugger;
+	private final LadybugReportGenerator reportGenerator;
 
-	private PipeLineSessionDebugger(PipeLineSession pipeLineSession, IbisDebugger ibisDebugger) {
+	private PipeLineSessionDebugger(PipeLineSession pipeLineSession, LadybugReportGenerator ibisDebugger) {
 		this.pipeLineSession = pipeLineSession;
-		this.ibisDebugger = ibisDebugger;
+		this.reportGenerator = ibisDebugger;
 	}
 
-	public static PipeLineSession newInstance(PipeLineSession pipeLineSession, IbisDebugger ibisDebugger) throws NoSuchMethodException, IllegalArgumentException, InstantiationException, IllegalAccessException, InvocationTargetException {
+	public static PipeLineSession newInstance(PipeLineSession pipeLineSession, LadybugReportGenerator ibisDebugger) throws NoSuchMethodException, IllegalArgumentException, InstantiationException, IllegalAccessException, InvocationTargetException {
```

Comment:
Het is geen `ibisDebugger` meer, misschien kan je meteen deze variabele hernoemen?

Reply:
Ah, ik dacht dat deze via Spring gewired werd (byName) maar dat is helemaal niet zo :) ik zal het aanpassen.